editor/import/resource_importer_texture.cpp

```diff
@@ -408,9 +407,10 @@ Error ResourceImporterTexture::import(const String &p_source_file, const String
 	const bool fix_alpha_border = p_options["process/fix_alpha_border"];
 	const bool premult_alpha = p_options["process/premult_alpha"];
 	const bool normal_map_invert_y = p_options["process/normal_map_invert_y"];
-	const bool stream = p_options["compress/streamed"];
+	// Support for texture streaming is not implemented yet.
+	const bool stream = false;
 	const int size_limit = p_options["process/size_limit"];
-	const bool hdr_as_srgb = p_options["process/hdr_as_srgb"];
+	const bool hdr_as_srgb = p_options["process/HDR_as_SRGB"];
```

Comment:
This captilization change is unwanted.

Reply:
That was a leftover from a rebase, fixed :slightly_smiling_face: